ReactWindows/ReactNative.Net46/Views/Image/ReactImageManager.cs

```diff
@@ -274,7 +274,10 @@ private void OnImageStatusUpdate(Border view, ImageStatusEventData status)
         /// </summary>
         /// <param name="view">The image view instance.</param>
         /// <param name="source">The source URI.</param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")]
+#pragma warning disable AvoidAsyncVoid
         private async void SetUriFromSingleSource(Border view, string source)
```

Comment:
Hey @rozele, I suppressed `AvoidAsyncVoid` on `SetUriFromSingleSource`, but would we want this method to be `async` without `await` operator?